src/swsssdk/config/database_global.json

```diff
@@ -0,0 +1,20 @@
+{
+    "INCLUDES" : [
```

Comment:
if this is for unit test, then it should be moved to tests/config folder

Reply:
Moved the file to tests/config